lib/Target/RISCV/MCTargetDesc/RISCVMCTargetDesc.h

```diff
@@ -45,6 +45,10 @@ MCAsmBackend *createRISCVAsmBackend(const Target &T, const MCRegisterInfo &MRI,
 
 MCObjectWriter *createRISCVELFObjectWriter(raw_pwrite_stream &OS, uint8_t OSABI,
                                            bool Is64Bit);
+namespace RISCV_MC {
```

Comment:
This namespace name doesn't match LLVM conventions. This function also isn't really specific to the MC layer either (RISCVSubtarget could also make use of it, for instance).

Reply:
Modify the namespace to "Feature". Please let me know if you have better naming suggestion.